tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
We are only testing one sentence in that test case. And the dictionary for that sentence is sufficient.